tools/gcpviz/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM golang:1.14 AS builder
+
+WORKDIR /go/src/github.com/GoogleCloudPlatform/professional-services/tools/gcpviz
+COPY go.mod go.sum *.go ./
+COPY cmd/*.go ./cmd/
+RUN cd cmd
+RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -o $GOPATH/bin ./...
+
+FROM google/cloud-sdk:slim
```

Comment:
What's the needs for the cloud-sdk image ? Do you have a dependency on gcloud ? Otherwise can you use launcher.gcr.io/google/debian9:latest ?

Reply:
Exactly, there's a dependency on gcloud tool for running the CAI export.

Reply:
Aha i missed it sorry. Looks good ;)